spec/unit/puppet/type/dsc_xfirewall_spec.rb

```diff
@@ -228,42 +228,42 @@
 
   it 'should accept dsc_profile predefined value Any' do
     dsc_xfirewall[:dsc_profile] = 'Any'
-    expect(dsc_xfirewall[:dsc_profile]).to eq('Any')
+    expect(dsc_xfirewall[:dsc_profile]).to eq(['Any'])
   end
```

Comment:
:+1: 

Existing code generated test sets an array on our resource, while these new tests set a single string and ensure that we expose an array:


Existing test:

```ruby
 it 'should accept array of predefined values for dsc_profile' do
  dsc_xfirewall[:dsc_profile] = ["Any", "Public", "Private", "Domain"]
  expect(dsc_xfirewall[:dsc_profile]).to eq(["Any", "Public", "Private", "Domain"])
end
```